caom2pipe/manage_composable.py

```diff
@@ -2655,6 +2648,8 @@ def make_time_tz(from_value):
         result = None
         if temp is not None:
             result = datetime.fromtimestamp(temp, tz=timezone.utc)
+    elif isinstance(from_value, float):
+        result = datetime.fromtimestamp(from_value, tz=timezone.utc)
     else:
         result = from_value.fromtimestamp(from_value, tz=timezone.utc)
```

Comment:
What representation of time is this? Function docs could specify the expected types for this argument.

Reply:
Added function comments.